plugin.video.bfi/addon.xml

```diff
@@ -0,0 +1,36 @@
+<?xml version="1.0" encoding="UTF-8" standalone="yes"?>
+<addon id="plugin.video.bfi" name="BFI Player" version="1.2.0" provider-name="fraser">
+    <requires>
+        <import addon="xbmc.python" version="2.25.0"/>
```

Comment:
> WARN: For leia it is advised to set xbmc.python version to 2.26.0

Reply:
Ah my bad - will update that now.  Thanks for the catch!